COLLADABaseUtils/include/COLLADABUStringUtils.h

```diff
@@ -108,6 +127,8 @@ namespace COLLADABU
 		static char toUpperASCIIChar( char c );
 
         static String uriEncode ( const String & sSrc );
+		static String ucs2Encode(const String & sSrc);
```

Comment:
We're not encoding URIs here, we're encoding XML names.
That's not the same encoding/decoding scheme at all.